test/extended/util/disruption/frontends/known_backends.go

```diff
@@ -4,55 +4,109 @@ import (
 	"context"
 
 	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	routeclient "github.com/openshift/client-go/route/clientset/versioned"
 	"github.com/openshift/origin/pkg/monitor"
 	"github.com/openshift/origin/pkg/monitor/backenddisruption"
 	"github.com/openshift/origin/test/extended/util/cluster"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/discovery"
 	"k8s.io/client-go/rest"
 	e2e "k8s.io/kubernetes/test/e2e/framework"
 )
 
+const (
+	oauthRouteNamespace = "openshift-authentication"
+	oauthRouteName      = "oauth-openshift"
+)
+
 func StartAllIngressMonitoring(ctx context.Context, m monitor.Recorder, clusterConfig *rest.Config) error {
-	if err := createOAuthRouteAvailableWithNewConnections().StartEndpointMonitoring(ctx, m, nil); err != nil {
+	// Ingress monitoring checks for oauth and console routes to monitor healthz endpoints. Check availability
+	// before setting up any monitors.
+	available, err := isRouteAvailable(ctx, clusterConfig, oauthRouteNamespace, oauthRouteName)
```

Comment:
available -> routeAvailable
to align with configAvailable

Reply:
Done